src/main/java/org/ehcache/sizeof/ObjectGraphWalker.java

```diff
@@ -263,6 +263,14 @@ private static void nullSafeAdd(final Deque<Object> toVisit, final Object o) {
                         LOG.error("Security settings prevent Ehcache from accessing the subgraph beneath '{}'" +
                                   " - cache sizes may be underestimated as a result", field, e);
                         continue;
+                    } catch (RuntimeException e) {
```

Comment:
Right now this is swallowing thrown `RuntimeException` instances without logging anything and then adding an inaccessible field to the collection. I think it's going to be simpler (and cleaner) to just:

```java
} catch (RuntimeException e) {
  LOG.warn("The JVM is preventing Ehcache from accessing the subgraph beneath '{}'" +
           " - cache sizes may be underestimated as a result", field, e);
}
```